app/assets/javascripts/load_time.js

```diff
@@ -45,6 +45,7 @@ MoJLoadTime = {
 	},
 
 	sendEventToGoogle: function(pageName, label, loadTime) {
+		ga('create', 'UA-37377084-14');
```

Comment:
Hey Jav :) just checking that the `UA-` part of the ID doesn't need to change. Are Google keeping it, considering Universal Analytics is coming to an end?

Reply:
Yes for now we keep GA3 running until GA4 is matured enough.